mandatory/4-tax.js

```diff
@@ -4,9 +4,16 @@
   A business requires a program that calculates how much the price of a product is including sales tax
   Sales tax is 20% of the price of the product.
 */
-
-function calculateSalesTax() {}
-
+// price*product gives you the total price without tax.
+// 1 + salesTax gives you the price after tax.
+function calculateSalesTax(price,product) {
+  var salesTax = .20;
+  var totalPrice = (price * product) * (1+salesTax);
+  return "£"+totalPrice;
```

Comment:
I think there's some confusion what this function is supposed to do. There's no formatting that is supposed to happen here, you're simply to return the number. I.e. 
```suggestion
  return totalPrice;
```